docs/Reference/API-Methods.md

```diff
@@ -88,55 +88,107 @@ This example changes the debug level of all logs to `WARN`.
     ```bash tab="curl HTTP request"
     curl -X POST --data '{"jsonrpc":"2.0","method":"admin_changeLogLevel","params":["WARN"], "id":1}' http://127.0.0.1:8545
     ```
-    
+
     ```bash tab="wscat WS request"
     {"jsonrpc":"2.0","method":"admin_changeLogLevel","params":["WARN"], "id":1}
     ```
-    
+
     ```json tab="JSON result"
     {
      "jsonrpc": "2.0",
      "id": 1,
      "result": "Success"
     }
     ```
-    
+
+### admin_generateLogBloomCache
+
+Generates cached log bloom indexes for blocks. APIs such as [`eth_getLogs`](#eth_getlogs)
+and [`eth_getFilterLogs`](#eth_getfilterlogs) use the indexes for improved
+performance.  
+
+!!! note
+    Each index file contains 100000 blocks. The last fragment of blocks less that 100000 are not indexed.
+
+Must be manually executed to add new blocks to the indexes.
+
+**Parameters**
+
+`integer` - Block to start generating indexes
+
+`integer` - Block to stop generating indexes
+
+**Returns**
+
+`result` : *object* - Log bloom index details:
+
+* `integer` : `startBlock` - Starting block for the generated indexes
+* `integer` : `endBlock` - End block for the generated indexes
+* `quantity` : `currentBlock` - Hexadecimal integer representing the latest
```

Comment:
I think calling out the hexideicmal number is already taken care of by calling it a quantity.
```suggestion
* `quantity` : `currentBlock` - The most recent block that was added to the cache
```